### PR TITLE
add katherine to demo users

### PR DESCRIPTION
**Pull request by wkloucek**
## Description
adds katherine as a new user with space-manager permissions

## Related Issue
- requested by @michaelstingl 

## Motivation and Context
easily play around with spaces

### CODE REVIEW

**Comment by wkloucek** [comment]
@C0rby I couldn't get the create and list spaces permissions working. If I add them, katherine cannot login. Do you have an idea?

File: settings/pkg/service/v0/settings.go [hunk 1]
```diff
@@ -476,6 +475,18 @@ func defaultRoleAssignments() []*settingsmsg.UserRoleAssignment {
...
 			AccountUuid: "932b4540-8d16-481e-8ef4-588e4b6b151c",
 			RoleId:      BundleUUIDRoleUser,
+		}, {
+			AccountUuid: "534bb038-6f9d-4093-946f-133be61fa4e7",
+			RoleId:      BundleUUIDRoleUser,
+		},
+		// kjohnson with additional role "space-manager" (ListAllSpaces + CreateSpace)
+		{
+			AccountUuid: "534bb038-6f9d-4093-946f-133be61fa4e7",
+			RoleId:      ListAllSpacesPermissionID,
```
Comment (kulmann) [inline settings/pkg/service/v0/settings.go:485]: You're assigning a permission id as a role id. I guess you need to create a `SpaceManagers` role and use the role id here.

Comment (wkloucek) [reply settings/pkg/service/v0/settings.go]: thanks for that hint!

**Review by kulmann — APPROVED** [review]
LGTM 👍

**Pull request merged**